tests/tcp_hang_test/README

```diff
@@ -0,0 +1,43 @@
+Test Name:
+        TCP hang client-server
+
+Author(s):
+        Daniel Libenson danielli@mellanox.com
+
+Short description:
+	This is a client server test that built to reproduce a tcp hang issue 
```

Comment:
need better details about the "tcp hang"... besides the test is a test even if we solved the hang.
In this case:  This is a client server test that built to exercise the TCP window size changing during traffic (or something in that direction)